### DISCUSSION TITLE
Screen not working after turning Ultimate mode on

### DISCUSSION THREAD

**Discussion by ma7876** [seerge/g-helper#1816] [Q&A] [answered]
Hello, I turned on Ultimate mode on my laptop ASUS ROG STRIX SCAR 18 (G834JY-XS97) and after reboot screen is not turning on and second connected screen doesn't turn on as well. Please help I am currently sitting in front of laptop and have no idea what to do.

**Comment by ma7876** [comment, marked as answer]
Making force shut down and booting up into bio by spamming F2 worked. I existed bios and went back to system and display was working. I am not gonna risk anymore by using Ultimate mode.

**Comment by seerge** [comment]
@ma7876 some devices require you to set Standard mode prior to switching to Ultimate. I assume you switched to Ultimate directly from Eco ?

**Reply by ma7876** [reply]
Yes, I think that is what happened, I was in Eco mode before changing to Ultimate.

**Comment by seerge** [comment]
@ma7876 I have adde your model to the list, where app will automatically set Standard mode prior to rebooting to Ultimate
[GHelper.zip](https://github.com/seerge/g-helper/files/13797738/GHelper.zip)
